### ISSUE TITLE
Stats on ipv6 support

### ISSUE THREAD

**Issue by xiam** [getlantern/lantern#2557]
It would be very interesting to see an statistic on which percentage of Lantern users can reach ipv6 sites. Probably we just need to setup an ipv6-only DNS record, see if a client is able to reach it and the log the result for posterior analysis.

**Comment by myleshorton** [comment]
Agreed @xiam! Just added to beta 7, but feel free to start working on this any time.
